frontend/src/App.js

```diff
@@ -1,41 +1,46 @@
-import { BrowserRouter, Routes, Route } from "react-router-dom";
-import Signup from "./pages/Signup";
-import Login from "./pages/Login";
-import PageNotFound from "./pages/PageNotFound";
-import Home from "./pages/Home";
-import UserTokenTest from "./pages/UserTokenTest";
-import Layout from "./components/Layout";
-import RequireAuth from "./components/RequireAuth";
-import Profile from "./pages/Profile";
-import PersistLogin from './components/PersistLogin';
+import { BrowserRouter, Routes, Route } from 'react-router-dom'
+import Signup from './pages/Signup'
+import Login from './pages/Login'
+import PageNotFound from './pages/PageNotFound'
+import Home from './pages/Home'
+import UserTokenTest from './pages/UserTokenTest'
+import Layout from './components/Layout'
+import RequireAuth from './components/RequireAuth'
+import Profile from './pages/Profile'
+import PersistLogin from './components/PersistLogin'
+import Match from './pages/Match'
+import Room from './pages/Room'
+
 function App() {
-    return (
-        <div className="App">
-            <BrowserRouter>
-                <Routes>
-                    <Route path="/" element={<Layout />}>
-                        {/* public routes */}
-                        <Route element={<PersistLogin />}>
-                            <Route path="/signup" element={<Signup />} />
-                            <Route path="/login" element={<Login />} />
-                        </Route>
+  return (

```

Comment:
Likely there's some conflict in our default prettier config/space choice, will need to fix the styling setup soon to avoid unnecessary file changes